healthcareai/tests/test_oversampling.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+import pandas as pd
+import numpy as np
+
+import healthcareai as hc
+
+
+class TestOverSampling(unittest.TestCase):
+    def setUp(self):
+        self.df = pd.DataFrame([
+                  [1.3, 3.4, 'one'],
+                  [1.5, 3.3, 'one'],
+                  [1.4, 3.5, 'one'],
+                  [1.4, 3.5, 'two'],
+                  [1.7, 3.6, 'one'],
+                  [1.2, 3.5, 'three'],            
+                  [1.7, 3.6, 'two']  ])
+        self.df.columns = ['x1','x2','y']
+        
+    def test_number_of_rows_returned(self):
+        expected_rows = max(self.df.y.value_counts()) * len(self.df.y.unique())
+        oo = hc.DevelopSupervisedModel(dataframe = self.df,
+                                       model_type = 'classification',
+                                       predicted_column = 'y')
+        oo.over_sampling()
+        self.assertEqual(expected_rows, oo.dataframe.shape[0])
+
+    def test_returns_correct_type(self):
```

Comment:
Make the test name even more explicit - test_returns_dataframe or something like that